packages/components/src/number-control/test/index.tsx

```diff
@@ -101,7 +101,7 @@ describe( 'NumberControl', () => {
 			// Second call: type '1'
 			expect( onChangeSpy ).toHaveBeenNthCalledWith( 2, '1', false );
 			// Third call: clamp value
-			expect( onChangeSpy ).toHaveBeenNthCalledWith( 3, 4, true );
+			expect( onChangeSpy ).toHaveBeenNthCalledWith( 3, '4', true );
```

Comment:
Changes in this file reflect the bug fix related to `onChange`'s signature — with the changes in this PR, its argument is always a `string` (in line with existing documentation)